network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/TerminalImpl.java

```diff
@@ -42,8 +42,22 @@ public TerminalImpl(NetworkObjectIndex index, Connectable<?> connectable, Functi
         this.index = index;
         this.connectable = connectable;
         this.attributesGetter = attributesGetter;
-        nodeBreakerView = new TerminalNodeBreakerViewImpl<>(index, connectable, attributesGetter);
-        busBreakerView = new TerminalBusBreakerViewImpl<>(index, connectable, attributesGetter);
+        nodeBreakerView = new TerminalNodeBreakerViewImpl<>(index, connectable, attributesGetter) {
+            @Override
+            public void moveConnectable(int node, String voltageLevelId) {
+                TopologyPoint oldTopologyPoint = TerminalImpl.this.getTopologyPoint();
+                super.moveConnectable(node, voltageLevelId);
+                index.notifyUpdate(connectable, "moveConnectable", oldTopologyPoint, TerminalImpl.this.getTopologyPoint());
```

Comment:
attribute name "moveConnectable" is not the same as the one in powsybl-core ... (where it is : "terminal" + side)

 notifyUpdate("terminal" + (iSide + 1), oldTopologyPoint, terminalExt.getTopologyPoint());

Reply:
Done